### DISCUSSION TITLE
Fresh Install - Register -> 403 Error - Forbidden

### DISCUSSION THREAD

**Discussion by pheonix-devapps** [firefly-iii/firefly-iii#8092] [Firefly III]
Forbidden
You don't have permission to access this resource.

All folders and files in "storage" & "public" folders are set to 775

If you need the website for details contact me directly.

**Comment by JC5** [comment]
Probably web server configuration?

**Reply by pheonix-devapps** [reply]
@JC5 [](url)

**Reply by JC5** [reply]
The error is not from Firefly III I'm afraid. Is there anything in the logs?

**Reply by pheonix-devapps** [reply]
how to generate or initiate debug logs???

**Reply by pheonix-devapps** [reply]
ANY ALTERNATE WAY TO CREATE USER FROM CLI OR DB DIRECTLY?

**Reply by JC5** [reply]
You haven't yet shared how you installed Firefly III, could you be a bit more specific?

**Reply by pheonix-devapps** [reply]
I have installed through composer and it's commands from ssh access to my shared hosting server.

**Reply by pheonix-devapps** [reply]
[2023-10-25 14:29:22] local.DEBUG: Now in Handler::render()  
[2023-10-25 14:29:22] local.DEBUG: Error "Illuminate\Auth\AuthenticationException" has no Firefly III treatment, parent will handle.

**Reply by JC5** [reply]
That's an interesting error, but this is just before you get redirected to the login form. What do the webserver logs tell you?

**Reply by pheonix-devapps** [reply]
I believe if give me way to create id through cli or sql query,  I would look up for further error.

**Reply by pheonix-devapps** [reply]
website**** 172.69.166.33 - - [23/Oct/2023:19:41:56 +0100] "GET /login HTTP/1.1" 302 394 "-" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Mobile Safari/537.36"
wallet. 172.69.166.33 - - [23/Oct/2023:19:41:57 +0100] "GET /register HTTP/1.1" 200 6863 "-" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Mobile Safari/537.36"
wallet. 172.70.143.112 - - [23/Oct/2023:19:42:00 +0100] "GET /v3-local/logo/logo-session.png HTTP/1.1" 200 12753 "-" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Mobile Safari/537.36"
wallet.b 172.70.142.219 - - [23/Oct/2023:19:42:02 +0100] "GET /v3-local/lib/fontawesome-free/webfonts/fa-solid-900.woff2 HTTP/1.1" 200 78268 "https://wallet.xxxx.co.in/v3-local/css,_fonts.css,qv==6.0.23+lib,_fontawesome-free,_css,_all.min.css,qv==6.0.23+lib,_icheck-bootstrap,_icheck-bootstrap.min.css,qv==6.0.23+dist,_css,_adminlte.min.css,qv==6.0.23.pagespeed.cc.3l0EuoVotK.css" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0

**Reply by pheonix-devapps** [reply]
[23/Oct/2023:19:42:18 +0100] "POST /register HTTP/1.1" 403 199 "-" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Mobile Safari/537.36"

**Reply by JC5** [reply]
Like I said, Firefly III is not the cause of the 403. I also see that you run a pagespeed module. The answer may be in that direction.
